README.md

```diff
@@ -1,25 +1,211 @@
 [![codecov](https://codecov.io/gh/chanzuckerberg/cell-census/branch/main/graph/badge.svg?token=byX1pyDlc9)](https://codecov.io/gh/chanzuckerberg/cell-census)
 
-# CELLxGENE Cell Census
+# Cell Census of CZ CELLxGENE Discover
```

Comment:
A user who wants to see what the census can do has to scroll to find an explanation. I like that the documentation for [scanpy](https://scanpy.readthedocs.io/en/stable/) and [seurat](https://satijalab.org/seurat/index.html) have a very prominent link within the first screen of the page to vignettes and quick start. I'd suggest a set of links in the header section as follows: 

Get started: [Tutorials](https://github.com/chanzuckerberg/cell-census/tree/main/api/python/notebooks), Quick Start (link to "Getting started" header in the readme)